### DISCUSSION TITLE
Happy New Year!

### DISCUSSION THREAD

**Discussion by louthy** [louthy/language-ext#971] [Announcements]
Happy new year all, here’s hoping for a brighter 2022 :)

I’m going to be a little quiet in Q1, as I need to reclaim a bit of personal time.  I’ll still respond to issues and prs as quickly as I can obviously.

Paul 👌

**Comment by seungyongshim** [comment]
Thank you for your honorable works.
ps) I'm waiting echo-process for Aff....
